lib/net.js

```diff
@@ -133,6 +134,7 @@ function Socket(options) {
   this._handle = null;
   this._parent = null;
   this._host = null;
+  this._timeout = null;
```

Comment:
It gets added if Socket#setTimeout() is called iirc https://github.com/nodejs/node/blob/master/lib/net.js#L330